Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/WebsiteProjectController.java

```diff
@@ -85,6 +105,19 @@ public String showProject(Model model, @PathVariable("ProjectUnixName") String u
         if (principal != null) {
             user = principal.getName();
         }
```

Comment:
why do you need the user here? You don't seem to use it in the method, or am I missing something?